scripts/network-init

```diff
@@ -108,21 +108,14 @@ rename_network_label() {
     done
 }
 
-maybe-enable-tls-verification() {
-  verification_enabled="$(${XE} pool-list params=tls-verification-enabled --minimal)"
-  if [ "$FIRSTBOOT_ENABLE_TLS_VERIFICATION" == "true" ] && [ "$verification_enabled" != "true" ] ; then
-      ${XE} pool-enable-tls-verification
-  fi
-}
-
 if [ "$UPGRADE" = "true" ]; then
     rename_network_label
 else
     prepare_networking
     rename_network_label
     # Ensure changes are synced to disk
     xe pool-sync-database
-    maybe-enable-tls-verification
+    ${XE} pool-enable-tls-verification
```

Comment:
We don't need to check anymore if it is already enabled, as in the old function?

Reply:
The call exchanges certificates every time. 

I think this behaviour is desired to be able to recover pools, as it should be idempotent, although it may do additional non-essential work.

We may want to reintroduce the check here

Reply:
@lindig ?

Reply:
What is the question? About the behaviour when certs already exist and are distributed? We have a design where verification can be enabled once and disabled per host in an emergency. Looking at the code I don't immediately see that it would refresh certs.

Reply:
Mh, it might not be needed, first boot is always a single-host pool